### PR TITLE
Collection of OSS project cc article

### LINKED ISSUE #2004: A collection of guidance and templates for open-source software projects
https://github.com/corsa-center/oss-documents/

### PR DESCRIPTION
**Pull request by pagrubel**
# Add a cc article for the Collection of OSS project repository

EB Member: @ksbeattie 
Author: @pagrubel 

Resolves #2004 

## PR checklist for files displayed on bssw.io site

* [x] ***[Author]*** `@mention` the BSSw.io editorial board member `@<eb-member-id>` in **Description** above assigned to shepherd your PR.
* [x] ***[Author]*** Add the `<issue-id>` in the **Description** above for the associated GitHub Issue.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py])~~.
* [x] ***[Author]*** Inspect the content in the `*.md` file(s) as rendered in GitHub for this PR.
* [x] ***[EB Mem]*** Assign this PR to the EB member `<eb-member-id>`.
* [x] ***[EB Mem]*** Assign this PR to the author of the PR `<pr-author-id>`.
* [x] ***[EB Mem]*** Add one or more Reviewers.
* [x] ***[EB Mem]*** Add label `content: <content-type>` for the type of contribution.
* [x] ***[EB Mem]*** Add to Project `Content Development` (see [Content Development]).
* [x] ***[EB Mem]*** Add [meta-data] to the `*.md` file(s) (set `Publish: yes`).
* [x] ***[EB Mem]*** Add label `preview` (so PR branch will be merged to 'preview' branch and watch for possible merge failures).
* [x] ***[EB Mem]*** Rebuild [preview] site and confirm new content is there, renders correctly and is returned in searches.
* [x] ***[Author]*** Make any final changes to the PR based on feedback.
* ~~[ ] ***[Author]*** Ensure `wikize_refs.py -i <base>.md` is run and commit (if using [wikize_refs.py]).~~
* [x] ***[EB Mem]*** Rebuild [preview] site and re-confirm content looks correct.
* [x] ***[EB Mem]*** Ensure at least one reviewer signs off on the final changes.
* [x] ***[EB Mem]*** Move the PR to "Ready to Publish" in [Content Development].
* [x] ***[EB Mem]*** Leave comment and @mention `@betterscientificsoftware/bssw-maint` (**BSSw Maint**) asking to carry out final publication steps.
* [x] ***[BSSw Maint]*** Verify that all needed files are present in the PR (article, images, updates to Site/Homepage.md carousel and/or Site/Announcements/Announcements.md as appropriate).
* [x] ***[BSSw Maint]*** Merge PR. (Should automatically move to "Done" in [Content Development].)
* [ ] ***[BSSw Maint]*** Verify new contribution shows up on [bssw.io] as expected.

NOTE:
* Checklist items prefixed with ***[Author]*** are expected to be performed by the author of the PR or can be performed by the author.
* Checklist items prefixed with ***[EB Mem]*** must be performed by a [BSSw.io Editorial Board (EB) Member](https://betterscientificsoftware.github.io/bssw.io/bssw_members.html).
* Checklist items prefixed with ***[BSSw Maint]*** must be performed by someone in the `@betterscientificsoftware/bssw-maint` team (hint: type `@`, `b`, `s`, `s`, `w`, `-`, `m`  to auto-complete to `@betterscientificsoftware/bssw-maint`).

[preview]: https://preview.bssw.io
[bssw.io]: https://bssw.io
[Content Development]: https://github.com/betterscientificsoftware/bssw.io/projects/3
[BSSw Internal]: https://github.com/betterscientificsoftware/bssw.io/projects/2
[meta-data]: https://betterscientificsoftware.github.io/bssw.io/bssw_styling_common.html#metadata-section
[wikize_refs.py]: https://github.com/betterscientificsoftware/bssw.io/blob/master/utils/README.md#wikize_refspy

### CODE REVIEW

**Comment by pagrubel** [comment]
@ksbeattie I have the article up and on the preview site. It's rather sweet and short but if you think I should expand it let me know

File: CuratedContent/CollectionOpenSourceSwProjects.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+
+## A Collection of Guidance and Templates for Open-source Software (OSS) Projects 
+
+<!-- deck text start -->
+What better way to keep a collection of important documents to help OSS Projects than an open-source repository. The repository is a collection of documents for guidance, governance, examples, and templates for OSS projects. As with any open source project, contributions are encouraged.
```
Comment (ksbeattie) [inline CuratedContent/CollectionOpenSourceSwProjects.md:5]: What about something like the following slight change...
```suggestion
What better way to keep a collection of important documents to help OSS Projects than an open-source repository? Now there is such a repository with a collection of documents for guidance, governance, examples, and templates for new and existing OSS projects. And as with any open source project, contributions are encouraged.
```

**Review by ksbeattie — APPROVED** [review]
Minor suggested change, not required though.

**Comment by pagrubel** [comment]
@ksbeattie I actually made it a question since you added a question mark. It wasn't even a real sentence before.

**Pull request merged**